instances/bodySystem/vascularSystem.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/vascularSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "'Vascular system' is an anatomical system that consists of all blood and lymph vessels.",
```

Comment:
"The 'vascular system' is an anatomical organ system that consists of all vessels in the body, and carries blood and lymph through all parts of the body."